docs/UserGuide.md

```diff
@@ -3,18 +3,28 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Logging in](#login)
-- [Accessing the API](#apiAccess)
-- [Obtaining an API access token](#apiToken)
-- [Importing mismatches](#importing)
+- [Logging in](#logging-in)
+- [Accessing the API](#accessing-the-api)
+- [Obtaining an API access token](#obtaining-an-api-access-token)
+- [Importing mismatches](#importing-mismatches)
+  - [Creating a mismatches import file](#creating-a-mismatches-import-file)
+  - [Getting upload rights](#getting-upload-rights)
+  - [Uploading an import file](#uploading-an-import-file)
+    - [Headers](#headers)
```

Comment:
I wonder if we need to go to such a deep level, maybe we can run it with the `--maxlevel` flag set to 2, like so:

```
$ npx doctoc ./docs --maxlevel=2
```

That way the TOC won't be so busy

Reply:
yes, i was thinking if we need a maxlevel, i don't know what level of detail we want but yes we can do.

Reply:
@itamargiv `docs/UserGuide.md` would be better with --maxlevel=3 imho, what do you think?

```
- [Logging in](#logging-in)
- [Accessing the API](#accessing-the-api)
- [Obtaining an API access token](#obtaining-an-api-access-token)
- [Importing mismatches](#importing-mismatches)
  - [Creating a mismatches import file](#creating-a-mismatches-import-file)
  - [Getting upload rights](#getting-upload-rights)
  - [Uploading an import file](#uploading-an-import-file)
  - [Reviewing Mismatches](#reviewing-mismatches)
```

Reply:
yeah, that might be better